src/main/kotlin/gg/skytils/skytilsmod/mixins/hooks/renderer/RendererLivingEntityHook.kt

```diff
@@ -50,6 +51,17 @@ fun setColorMultiplier(
                 cir.returnValue = Skytils.config.seraphNormalPhaseColor.withAlpha(169)
             }
         }
+    } else if (Skytils.config.attunementDisplay && Utils.inSkyblock && entity is EntityLiving) {
+        (slayer as? SlayerFeatures.DemonlordSlayer)?.let {
+            if (entity == it.relevantEntity) {
+                entity.hurtTime = 0
+                it.relevantColor?.let {
+                    // Colors might be too hard to see because of the entities textures and colors,
```

Comment:
in this case, could try making alpha higher

Reply:
I have tried multiple alpha values for every attunement and different colors for Ashen specifically, but in the end this configuration seems to be the clearer since the alpha on the color multiplier doesn't quite translate to a clearer value. The animation when an entity is hurt uses 30% alpha in vanilla and it seems to be the clearest the color can get, which corresponds to 169 alpha on the hook. For Ashen I have tried the vanilla §7 and §0 but none of them look better than §8 in my opinion.